### ISSUE TITLE
How can we go about evaluating pre-distilled images this on Pytorch?

### ISSUE THREAD

**Issue by luke-mcdermott-mi** [yolky/RCIG#2]
I have not had luck using the pre-distilled images on general out of the box implementations of 128 wide conv models in pytorch.

**Comment by yolky** [comment]
Hi,
Apologies for the late response. What sorts of accuracies are you getting? Has the pre-distilled methods worked with our training script? Have you normalized the data in the same way? We are using ZCA normalization, so if you are using standard normalization, then our pre-distilled images will be incompatible.

**Comment by luke-mcdermott-mi** [comment]
Do you have a list of hyperparameters for evaluating somewhere? So, I need to use ZCA normalization the standard datasets, is there any crop/flip/other augmentations? Where can I find learning rate, epochs, batch size, optimizer other than scanning the whole repo for each

**Comment by luke-mcdermott-mi** [comment]
I am using your code now and getting results with 

```python3 eval.py --dataset_name cifar10 --checkpoint_path ./distilled_images_final/0/cifar10_10/checkpoint_10000 --config_path ./configs_final/depth_3.txt --random_seed 0```

However, I am trying to get the reported results with ResNet. by switching the eval_arch to resnet, but not achieving anything above 20% accuracy. I have used depth_3.txt and depth_3_no_flip.txt.

In the supplemental section, you state that no data augmentation gets better results for cifar10; however, your configs for cifar10 use data augmentation. Why is this? Furthermore, what hyperparameters and I supposed to use for ResNet on cifar10? I was assumeing its still depth_3.txt
